### DISCUSSION TITLE
Is it possible to connect to several VPNs from one Gluetun container, and to switch VPNs at runtime?

### DISCUSSION THREAD

**Discussion by d4h0** [qdm12/gluetun#990] [Ideas]
Hi,

Let's say, I'd like to open 3 different VPN connection (one for my browser, one for my email client, and one for other use), and I'd like to change the VPN every 30 minutes – is that somehow possible with Gluetun?

I couldn't find anything that indicates that any of that is possible, but I thought that I may miss something.

Gluetun looks pretty nice, thanks for creating it!

**Comment by qdm12** [comment]
Not possible now, but subscribe to #91 you should be able to modify any settings at runtime without a container restart. Although this is a few months away I think, I'm quite busy currently.

Note you can run 3 Gluetun on the same machine as well if that can help you solve your problem.

**Reply by d4h0** [reply]
Okay, thanks for answering my question, @qdm12!
